packages/xo-server-sdn-controller/src/ovsdb-client.js

```diff
@@ -57,16 +58,18 @@ export class OvsdbClient {
     const adding = { id: networkUuid, addr: remoteAddress }
     this._adding.push(adding)
 
-    const socket = await this._connect()
     const index = ++this._numberOfPortAndInterface
+    const socket = await this._connect()
```

Comment:
Why this change?

Reply:
To lock the index before the first `await`.

Reply:
Why is this necessary?

Why not only do this when you need to use the index.

Reply:
I've moved  `index` affectation where it's used.